src/lib/components/Circos.react.js

```diff
@@ -173,32 +173,34 @@ export default class Circos extends Component {
 
     configCircos(layout, config, tracks, setProps) {
         this.circos.layout(layout, config);
-        tracks.forEach((track, index) => {
-            const {id, data, config, type} = track;
+        if (tracks) {
+            tracks.forEach((track, index) => {
+                const {id, data, config, type} = track;
 
-            // Since config is const, can't manipulate and throws error
-            let configApply;
+                // Since config is const, can't manipulate and throws error
+                let configApply;
 
-            if (typeof config !== 'undefined') {
-                configApply = config;
+                if (typeof config !== 'undefined') {
+                    configApply = config;
 
-                // Set Event Handling
-                if (setProps) {
-                    configApply.events = this.setEvent(setProps, index);
-                }
+                    // Set Event Handling
+                    if (setProps) {
```

Comment:
`setProps` check isn't necessary anymore

Reply:
`Ideogram.react.js` also does a `setProps` check. Either for now or as a subsequent clean up PR.

Reply:
Fixed in 03131de!